--- bcunit-5.3.26/Examples/CMakeLists.txt.orig	2024-03-04 21:56:31.251883864 +0100
+++ bcunit-5.3.26/Examples/CMakeLists.txt	2024-03-04 21:57:01.881717928 +0100
@@ -22,7 +22,7 @@
 
 add_library(bcunitexamples STATIC ExampleTests.c)
 
-include_directories(${CMAKE_CURRENT_SOURCE_DIR})
+include_directories(${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_BINARY_DIR}/BCUnit/Headers ${CMAKE_SOURCE_DIR}/BCUnit/Headers)
 
 if(ENABLE_BCUNIT_BASIC)
 	add_subdirectory(BasicTest)
