--- BCunit-5.2.0/BCUnit/Sources/CMakeLists.txt~	2017-07-20 16:06:38.000000000 +0200
+++ BCunit-5.2.0/BCUnit/Sources/CMakeLists.txt	2018-09-26 11:47:06.695729380 +0200
@@ -176,6 +176,7 @@ if(ENABLE_STATIC)
 endif()
 if(ENABLE_SHARED)
 	add_library(bcunit SHARED ${SOURCE_FILES})
+	set_target_properties(bcunit PROPERTIES SOVERSION 1)
 	set_target_properties(bcunit PROPERTIES VERSION 1.0.1)
 	target_compile_definitions(bcunit INTERFACE "$<BUILD_INTERFACE:IN_BCUNIT_SOURCES>")
 	target_include_directories(bcunit PUBLIC
